lib/containers/basetest.pm

```diff
@@ -7,6 +7,8 @@
 # Maintainer: qac team <qa-c@suse.de>
 
 package containers::basetest;
+use containers::docker;
+use containers::podman;
 use Mojo::Base 'opensusebasetest';
 use containers::engine;
```

Comment:
@asmorodskyi I got a notification from you about the need of this line, but can't find it as comment here. Actually good point!

Reply:
because I deleted it :D . after writing it I realized that `containers::docker` and `containers::podman` are children of `containers::engine` so I wrongly conclude that it actually needed :D